src/services/user/forcePasswordChange.js

```diff
@@ -0,0 +1,35 @@
+const { checkPasswordStrength, passwordsMatch } = require('../../helper/passwordHelpers');
+const { BadRequest } = require('@feathersjs/errors');
```

Comment:
Codacy found an issue: [`@feathersjs/errors` import should occur before import of `../../helper/passwordHelpers`](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5748939)